### ISSUE TITLE
Post install fails

### ISSUE THREAD

**Issue by EWhite613** [jvilk/mitmproxy-node#3]
```
> mitmproxy@2.0.4 postinstall /Users/ewhite/.nvm/versions/node/v10.7.0/lib/node_modules/mitmproxy
> scripts/install_python_deps.py

Traceback (most recent call last):
  File "scripts/install_python_deps.py", line 18, in <module>
    version[2] = int(version[2])
ValueError: invalid literal for int() with base 10: '4\\nPython:'
```
Looks like format given isn't expected
```
mitmdump --version
Mitmproxy: 4.0.4
Python:    3.7.0
OpenSSL:   OpenSSL 1.0.2o  27 Mar 2018
Platform:  Darwin-16.7.0-x86_64-i386-64bit
```

**Comment by epatrasc** [comment]
I have the same problem.

I have tried to run manually the script 'install_python_deps.py' and seems to works fine but when I run 'npm install -g bleak-detector'  it fails.

**Comment by lukestoward** [comment]
I also see this when installing via: `npm install -g mitmproxy@2.0.4` with the postinstall call.

**Comment by jvilk** [comment]
Sorry for the delay in responding; I just defended my PhD and became Dr. Vilk, and am in the process of moving across the US!

Thanks for the notes, everyone. I thought I fixed this bug in v2.0.4, and I need to update BLeak's dependency on mitmproxy. I wish I didn't have to do something so fragile to get a version number, but checking the version number is important to verify that the package will work correctly. I designed the package such that if it installs, it should *just work*.

@Stoward94 what output do you get when you run `mitmdump --version`? I'd like to know if there are other variants I need to parse.

**Comment by lukestoward** [comment]
@jvilk wow incredible news, congratulations!

I get the following from `mitmdump --version`
```
Mitmproxy: 4.0.4
Python:    3.7.0
OpenSSL:   OpenSSL 1.0.2o  27 Mar 2018
Platform:  Darwin-17.7.0-x86_64-i386-64bit
```

**Comment by digitarald** [comment]
Congratulations Dr. @jvilk (and thank you finding the leak in debugger-html 😄) !

Same here, freshly installed mitmproxy via pip3. `yarn global add bleak-detector` fails with the error similar to comment 1.

```
Mitmproxy: 4.0.4
Python:    3.7.0
OpenSSL:   OpenSSL 1.1.0h  27 Mar 2018
Platform:  Darwin-17.7.0-x86_64-i386-64bit
```

**Comment by jvilk** [comment]
Thanks for your patience. I pushed a new version with the fix (2.1.0), and updated BLeak to use the new version.

Please let me know if you continue to have problems with the new version.

**Comment by epatrasc** [comment]
Now it works fine, thanks!
